mlrun/common/helpers.py

```diff
@@ -14,9 +14,9 @@
 
 
 def parse_versioned_object_uri(
-    uri: str, default_project: str = ""
+    uri: str, active_project: str = ""
 ) -> tuple[str, str, str, str]:
-    project = default_project
+    project = active_project
```

Comment:
In this case it should remain `default_project`, because the helper function defaults to the project in the argument if not found in the URI. Unrelated to the config's `default_project`.